docs/output.md

```diff
@@ -60,14 +60,25 @@ PEPTIDE contains information about peptide hits that were identified and corresp
 <details markdown="1">
 <summary>Output files</summary>
 
-- `Intermediate_Results/`
-  - `*merged_psm_perc_filtered.mzTab` : If `--refine_fdr_on_predicted_subset` is specified, consists of the hits (filtered by q-value)
-  - `*.mztab` : mztab file generated by the OpenMS MzTabExporter command, the community standard format for sharing mass spectrometry search results
-  - `*.featureXML` : If `--skip_quantification` is not specified, then this file is generated by the OpenMS FeatureFinderIdentification command
-  - `*fdr_filtered.idXML`: If `--skip_quantification` is not specified, then this file is generated by the OpenMS IDFilter command
-  - `*all_ids_merged_psm_perc*.idXML`: idXML files are generated when `--refine_fdr_on_predicted_subset` is specified
-  - `*peptide_filtered.idXML`: If `--refine_fdr_on_predicted_subset` is specified, then this file consists of the PSMs prediction outcome
-  - `*perc_subset.idXML`: If `--refine_fdr_on_predicted_subset` is specified, then this file is the outcome of the second percolator run, generated by the OpenMS PercolatorAdapter
+- `intermediate_results/`
+  - `alignment`
+    - `*filtered.idXML` : If `--skip_quantification` is not specified, then this file is generated in the `OPENMS_IDFILTER_Q_VALUE`
+    - `{ID}_-_{filename}_filtered` : An outcome file of `OPENMS_IDFILTER_FOR_ALIGNMENT`, this file is only generated when `--skip_quantification` is not specified
+  - `comet`
+    - `{raw filename}.tsv` : The outcome of `CometAdapter` containing more detailed information about all of the hits that have been found (no filtering has been applied)
+    - `{Sample}_{Condition}_{ID}.tsv` : Single files that hold information about the peptides sequences that have been identified (no filtering has been applied)
+  - `features`
+    - `*.mztab` : mztab file generated by the OpenMS MzTabExporter command which is present in the `PROCESS_FEATURE` step
+    - `*.idXML` : Outcome of `PSMFEATUREEXTRACTOR`, containing the computations of extra features for each input PSM
+    - `*.featureXML` : These files file is generated by the OpenMS `FeatureFinderIdentification` command
+  - `percolator`
+    - `*all_ids_merged_psm_perc.idXML` : idXML files are generated with `OPENMS_PERCOLATORADAPTER`
+  - `refined_fdr` (Only if `--refine_fdr_on_predicted_subset` is specified)
+    - `*merged_psm_perc_filtered.mzTab` : This file export filtered percolator results (by q-value) as mztab
+    - `*_all_ids_merged.mzTab` : Exportas all of the psm results as mztab
+    - `*perc_subset.idXML` : This file is the outcome of a second OpenMS `PercolatorAdapter` run
```

Comment:
We run percolator multiple times :O?
Why?

Reply:
Percolator can be run two times when the parameters `--refine_fdr_on_predicted_subset` and `predict_class_1 ` are provided. Here, Percolator is used on the subset of predicted binders which results in an increased identification rate, hence the second iteration.